jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryClient.java

```diff
@@ -115,12 +116,9 @@ private String makeUserAgent() {
         return "";
       }
 
-      String version = RegistryClient.class.getPackage().getImplementationVersion();
       StringBuilder userAgentBuilder = new StringBuilder();
```

Comment:
Unrelated, but this looks like this User-Agent string doesn't really conform to the spec or customs. Assume the following doc is up to date, https://en.wikipedia.org/wiki/User_agent#Format_for_human-operated_web_browsers

But I admit it can be a pain to update the string, as backend systems may be relying on it.

Reply:
Should we create an issue for this?

Reply:
We use the string when interactive with registries, right? If so, let's just create it to keep track of it, even though it may take forever to get to this.

Reply:
#1067

Reply:
Thanks!